### PR TITLE
another docs update

### PR DESCRIPTION
**Pull request by dsikich**
- update conf.py for building latex documents
        - update build-intercept and start-stop console highlighting

### CODE REVIEW

File: docs/build-intercept.rst [hunk 1]
```diff
@@ -20,21 +22,21 @@ can find the libraries and headers it needs:
...
-	spack install environment-modules
-	spack load environment-modules
-	source <(spack module loads gotcha leveldb)
+	$ git clone https://github.com/spack/spack
+	$ . spack/share/spack/setup-env.sh
+	$ spack install leveldb
+	$ spack install gotcha
+	$ spack install environment-modules
+	$ spack load environment-modules
+	$ source <(spack module loads gotcha leveldb)
```
Comment (nedbass) [inline docs/build-intercept.rst]: While you're here, the above paragraph should begin "*To* install leveldb ..."

**Comment by adammoody** [comment]
@dsikich, in the interest of avoiding duplicated documentation, let's move the build and run related content from the main README.md file and add a link from README.md to our online docs.

**Comment by adammoody** [comment]
Let's also capture any useful info from the client/INSTALL and client/README files, so we can delete those from the repo.

**Comment by dsikich** [comment]
@adammoody @nedbass should that be part of this pull request? I was thinking I want to get everything settled into what should be on readthedocs, then delete any redundant information in a different update?

**Comment by dsikich** [comment]
@adammoody is there information that I missed from the client README and/or client/INSTALL that you are referring to that should be in build-intercept? There are a bunch of variables described in the client/README that I was on the fence about whether or not to add? Other than that I didn't see anything in client/INSTALL that is not contained on the website documentation

File: docs/build-intercept.rst (hunk 1 above)
Comment (dsikich) [reply docs/build-intercept.rst]: @nedbass thanks, this is updated now.

**Comment by nedbass** [comment]
> @adammoody @nedbass should that be part of this pull request? I was thinking I want to get everything settled into what should be on readthedocs, then delete any redundant information in a different update?

I think doing that in a separate pull request is fine.

**Comment by dsikich** [comment]
@adammoody @nedbass okay, I just wanted to make sure the content was ok as a first priority. I have one more update on this one. I added a link to the spack repo in build-intercept and I found a note about NUMA in client/INSTALL that I hadn't added.

**Pull request merged**